server/template.yml

```diff
@@ -181,42 +186,42 @@ Resources:
         Variables:
           SRC_BUCKET: !Ref SrcBucket
           JOB_TABLE: !Ref JobTable
-          POST_PROCESS_HANDLER_QUEUE: !Ref PostProcessHandlerQueue
+          POST_PROCESS_HANDLER_QUEUE: !Ref JobCompletionStatusQueue
     Metadata:
       Dockerfile: Dockerfile
       DockerContext: ./owlEye
       DockerTag: python3.7-v1
 
-  # FileCompress:
-  #   Type: AWS::Serverless::Function
-  #   Properties:
-  #     CodeUri: fileCompress/
-  #     Handler: app.lambda_handler
-  #     Runtime: python3.7
-  #     Timeout: 10
-  #     Policies:
-  #       # Give Create/Read/Update/Delete Permissions to the SampleTable
-  #       - DynamoDBCrudPolicy:
-  #           TableName: !Ref JobTable
-  #       - S3CrudPolicy:
-  #           BucketName: !Ref SrcBucket
-  #     Environment:
-  #       Variables:
-  #         # Make table name accessible as environment variable from function code during execution
-  #         JOB_TABLE: !Ref JobTable
-  #         SRC_BUCKET: !Ref SrcBucket
-  #     Events:
-  #       FileCompress:
-  #         Type: Api
-  #         Properties:
-  #           Path: /fileCompress
-  #           Method: get
+  FileCompress:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: fileCompress/
+      Handler: app.lambda_handler
+      Runtime: python3.7
+      Timeout: 10
+      Policies:
+        # Give Create/Read/Update/Delete Permissions to the SampleTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref JobTable
+        - S3CrudPolicy:
+            BucketName: !Ref SrcBucket
+      Environment:
+        Variables:
+          # Make table name accessible as environment variable from function code during execution
+          JOB_TABLE: !Ref JobTable
+          SRC_BUCKET: !Ref SrcBucket
+      Events:
+        FileCompress:
+          Type: Api
+          Properties:
+            Path: /fileCompress
+            Method: get
 
   SendEmail:
     Type: AWS::Serverless::Function
     Properties:
       CodeUri: sendEmail/
-      Handler: dist/send-email.sendEmail
+      Handler: dist/index.sendEmail
```

Comment:
is this a standard to have the name of file as index?

Reply:
Yep. I renamed your file, so it should be in the same place but called index.ts.